.changeset/thick-zoos-hug.md

```diff
@@ -0,0 +1,5 @@
+---
+'@crowdstrike/glide-core': patch
+---
+
+Resolved an issue with adding Checkboxes to Checkbox Group after the initial render.
```

Comment:
Writing about `privateMinimal` provides implementation details. This was the best I could come up with to describe how the fix resolves what the user was seeing. Open to suggestions.

Reply:
The only thing I'd suggest is to say what issue was resolved.